cypress/support/commands.js

```diff
@@ -0,0 +1,25 @@
+Cypress.Commands.add('login', (email, password) => {
+  const query = `
+        mutation{
+            loginAdmin(loginInput: { 
+                email: "${email}",
+                password: "${password}"
+            }){
+                token
+            }
+        }
+    `;
+
+  cy.request({
```

Comment:
Can you try to use `async await` here instead of `then` like syntax

Reply:
async/await constructions aren`t allowed inside of the cypress tests

Reply:
Okay